### PR TITLE
Sylvia/docs/doc 1247

### PR DESCRIPTION
**Pull request by srpearce**
@lamagnifica and @martynjames Please review.

### CODE REVIEW

**Comment by srpearce** [comment]
@lamagnifica @martynjames Aargh - did something wrong. Hold on a sec...

File: en_us/course_authors/source/developing_course/workflow.rst [hunk 1]
```diff
@@ -85,6 +85,26 @@ able to see it. Content visibility depends on several factors:
...
+
+Students can search course text in :ref:`HTML components<Working with HTML
+Components>` and video transcripts by using the **Search** box in the upper-left
+corner of the **Courseware** tab. 
+
+Before students can search your course, Studio must index the content. Studio
+indexes all new course content automatically when you :ref:`publish<Publish a
+Unit>` the content. You can also manually reindex all of the content in your
+course at any time. To do this, select **Reindex Content** in the upper- right
+corner of the **Course Outline** page.
```
Comment (lamagnifica) [inline en_us/course_authors/source/developing_course/workflow.rst]: spacing after the hyphen (damn you, ctrl-alt-q!)

File: en_us/course_authors/source/developing_course/workflow.rst [hunk 2]
```diff
@@ -85,6 +85,26 @@ able to see it. Content visibility depends on several factors:
   
 For more information, see :ref:`Controlling Content Visibility`.
 
+.. _Making Course Content Searchable:
+
+***********************************
+Making Course Content Searchable
+***********************************
```
Comment (lamagnifica) [inline en_us/course_authors/source/developing_course/workflow.rst:92]: start using "learner" instead of "student"?

**Comment by srpearce** [comment]
@lamagnifica @martynjames Long story short, I pushed to the Saudi doc branch instead of master (which is what I wanted to do, but I created my DOC-1247 branch off master, so there's a problem here). I'll get this sorted, but in the meantime, please review course_authors/source/developing_course/workflow.rst (new text only) and students/source/SFD_course_search.rst. Thanks!

File: en_us/course_authors/source/developing_course/workflow.rst [hunk 3]
```diff
@@ -85,6 +85,26 @@ able to see it. Content visibility depends on several factors:
...
+Components>` and video transcripts by using the **Search** box in the upper-left
+corner of the **Courseware** tab. 
+
+Before students can search your course, Studio must index the content. Studio
+indexes all new course content automatically when you :ref:`publish<Publish a
+Unit>` the content. You can also manually reindex all of the content in your
+course at any time. To do this, select **Reindex Content** in the upper- right
+corner of the **Course Outline** page.
+
+..note:: Typically, you would only manually reindex your course content
```
Comment (lamagnifica) [inline en_us/course_authors/source/developing_course/workflow.rst]: perhaps reorganize this note and the last 2 sentences into one paragraph? 
If necessary, you can manually reindex all of the content in your course. Typically, you only need to complete this procedure if students see unexpected search results. To reindex your course, select ... Reindexing usually takes less than ...

File: en_us/students/source/SFD_course_search.rst [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+#####################
+Searching the Course
+#####################
+
+Much of the content of your edX course is searchable, including course text and
+video transcripts. To search your edX course content, go to the **Courseware**
+tab, and then enter a search term in the **Search** field in the upper-left
+corner of the page.
+
+After you enter your search terms, your search results appear in the right
```
Comment (lamagnifica) [inline en_us/students/source/SFD_course_search.rst]: search terms --> search term

File: en_us/students/source/SFD_course_search.rst [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+pane. The search results include your search term, a small amount of the
```
Comment (lamagnifica) [inline en_us/students/source/SFD_course_search.rst]: a small amount --> (could you use the term "brief" instead? like, a brief excerpt from the context?)

File: en_us/students/source/SFD_course_search.rst [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+surrounding text, and the place in the course where the search term appears. To
+view your search term in the body of the course, click **View** to the right of
+the search result.
+
+Many searches return your search term and similar terms. For example, a search
+for "grade" returns results for "grades" and "graded".
+
+.. note:: You cannot use the **Search** field on the **Courseware** tab to 
+ search problems and course discussions. To search course discussions, use the
```
Comment (lamagnifica) [inline en_us/students/source/SFD_course_search.rst]: where do you search problems? Possibly, move the discussion-related info out of the note and leave problems by themselves as the outcasts...

**Comment by lamagnifica** [comment]
finished review, made a few suggestions

**Comment by srpearce** [comment]
@martynjames Have you had a chance to look at course_authors/source/developing_course/workflow.rst (new text only) and students/source/SFD_course_search.rst yet?

File: en_us/students/source/SFD_course_search.rst (hunk 6 above)
Comment (martynjames) [inline en_us/students/source/SFD_course_search.rst]: Perhaps separate the discussions from other "not indexed" content. Is there precedent for 2 notes?

Problems and other components may be added at a later time, don't know if it is worth mentioning this.

**Comment by martynjames** [comment]
@srpearce - all reads well to me - one comment on separating out note about discussions form note about other (non-indexed) content only -  LGTM :+1:

**Pull request merged**